docs/STANDALONE_MODE.md

```diff
@@ -0,0 +1,30 @@
+# Standalone mode
+
+Standalone mode enables plugin launching on different machine than Snap daemon (`snapteld`).
```

Comment:
Rather than `launching`, I think `start` would be a better word. Like this: 
```Stand-alone mode enables plugins to start and run on a different machine than Snap daemon (`snapteld`).```